File: GGAndroid_2.0/GGPay/pay-core/src/main/java/io/ganguo/pay/core/PayCallback.java
package io.ganguo.pay.core;

/**
 * Created by Roger on 05/07/2017.
 */

public interface PayCallback {
    void onPaySuccess(PayResult result);

    void onPayError(PayResult result);

    void onPayCancel(PayResult result);

    void onPayNotFound(String payType);
}

File: GGAndroid_4.0/core/service/service-pay/pay-core/src/main/kotlin/io/ganguo/pay/core/PayCallback.kt
package io.ganguo.pay.core

/**
 * Created by Roger on 05/07/2017.
 */

interface PayCallback<T : PayOrderInfo> {
    fun onPaySuccess(result: PayResult<T>)

    fun onPayError(result: PayResult<T>)

    fun onPayCancel(result: PayResult<T>)

}
